backend/gateway/gateway.go

```diff
@@ -39,7 +41,27 @@ type ComponentFactory struct {
 	Modules    module.Factory
 }
 
+func loadEnv(f *Flags) {
+	// Order is important as godotenv will NOT overwrite existing environment variables.
+	envFiles := f.EnvFiles
+
+	for _, filename := range envFiles {
+		// Use a temporary logger to parse the environment files
+		tmpLogger := newTmpLogger().With(zap.String("file", filename))
+
+		p, err := filepath.Abs(filename)
+		if err != nil {
+			tmpLogger.Fatal("parsing .env file failed", zap.Error(err))
+		}
+		// Ignore lint below as it is ok to to ignore dotenv loads as not all env files are guaranteed
+		// to be present.
+		// nolint
+		godotenv.Load(p)
```

Comment:
I think instead of failing silently we should log a warning at the very least that we weren't able to load said env files, what do you think?

Reply:
Yeah, I originally had that when testing locally but it's kind of noisy when starting the app if you don't have the files. I was trying to follow the react-scripts approach where they don't emit anything.

Reply:
Talking offline we decided to log on successful loads of env files, that why at a glance its easy to know whats been loaded.